src/providers/vertex.ts

```diff
@@ -184,13 +184,14 @@ export class VertexChatProvider extends VertexGenericProvider {
 
   async callGeminiApi(prompt: string, context?: CallApiContextParams): Promise<ProviderResponse> {
     // https://cloud.google.com/vertex-ai/docs/generative-ai/model-reference/gemini#gemini-pro
-    let contents = parseChatPrompt(prompt, {
+    let contents: any = parseChatPrompt(prompt, {
```

Comment:
parseChatPrompt takes a type `parseChatPrompt<{...}>`

probably should be a GeminiFormat in there